http/resource.go

```diff
@@ -144,6 +148,27 @@ var resourcePatchHandler = withUser(func(w http.ResponseWriter, r *http.Request,
 		return http.StatusForbidden, nil
 	}
 
+	override := r.URL.Query().Get("override") == "true"
+	rename := r.URL.Query().Get("rename") == "true"
+
+	if !override && !rename {
+		if _, err = d.user.Fs.Stat(dst); err == nil {
+			return http.StatusConflict, nil
```

Comment:
Error must be triggered only if src != dst

Reply:
This verification is required when duplicating files in the same folder, in a conflict scenario the frontend need to inform that the files should be renamed.